int32 decision_state
float32 angular_error